bin/build-coal-image

```diff
@@ -48,7 +50,7 @@ if [[ -z "$1" ]]; then
     usage 'must provide <usb_tarball> filename'
 fi
 
-TAR_BUILD_FILENAME=$(rel2abs $1)
+TAR_BUILD_FILENAME=$(rel2abs "$1")
```

Comment:
Do you need to quote around the $() like you do on line 12, e.g. `"$(rel2abs "$1")"` ?  (You might not, but I'm curious.)